esphome/components/abbaurora/__init__.py

```diff
@@ -0,0 +1,52 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+
+from esphome.components import uart
+from esphome.cpp_helpers import gpio_pin_expression
+from esphome.const import (
+    CONF_FLOW_CONTROL_PIN,
+    CONF_ID,
+    CONF_ADDRESS,
+)
+from esphome import pins
+
+CODEOWNERS = ["@ajvdw"]
+
+DEPENDENCIES = ["uart"]
+
+AUTO_LOAD = ["sensor", "text_sensor"]
+
+CONF_ABBAURORA_ID = "abbaurora_id"
+
+abbaurora_ns = cg.esphome_ns.namespace("esphome::abbaurora")
+ABBAurora = abbaurora_ns.class_("ABBAuroraComponent", uart.UARTDevice, cg.Component)
```

Comment:
```suggestion
ABBAuroraComponent = abbaurora_ns.class_("ABBAuroraComponent", uart.UARTDevice, cg.Component)
```

var name should match class name